install/gcp/gcp_installer.py

```diff
@@ -62,6 +62,8 @@ def run():
     parser.add_argument('--no-cloudshell',
                         action='store_true',
                         help='Bypass Cloud Shell requirement')
+    parser.add_argument('--service-account-key-path',
+                        help='Absolute path for service account key file')
```

Comment:
Can you please add into the help, that the filename is also needed?  Something like:
```
help='Absolute path and filename for service account key file'
```

Reply:
Done